.gitignore

```diff
@@ -198,3 +198,26 @@ lcov.info
 # /developer/src/test/auto/kmcomp/*.kvk
 # /developer/src/test/auto/kmcomp/*.kvk*
 # /developer/src/test/auto/kmcomp/*.txt
+windows/src/engine/keyman32/exports.h
+.gitignore
```

Comment:
This doesn't need to be here -- .gitignore is always ignored. Also would prefer to have the file-by-file exclusions under windows/src rather than top-level.